docs/react-v9/contributing/rfcs/react-components/fonts-segoe-sans-vs-segoe-ui.md

```diff
@@ -0,0 +1,85 @@
+# RFC: Fonts in v9: Segoe Sans vs Segoe UI
+
+<!--
+An RFC can be anything. A question, a suggestion, a plan. The purpose of this template is to give some structure to help folks write successful RFCs. However, don't feel constrained by this template; use your best judgement.
+
+Tips for writing a successful RFC:
+
+- Favor simple language that make your point, fancy words obfuscate // Speak plainly.
+- Try to stay concise, but don't gloss over important details
+- Feel free to add necessary diagrams in the RFC assets folder
+- Try to write a neutral problem statement, not one that motivates your desired solution
+- Remember, "Writing is thinking". It's natural to realize new ideas while writing your proposal
+-->
+
+---
+
+[@emmayjiang](https://github.com/emmayjiang)
+
+<!-- If substantial updates are made add an "Updated on: $date" below, don't replace the original date -->
+
+## Summary
+
+<!-- Explain the proposed change -->
+
+This proposal of this RFC is to change the font from Segoe UI to Segoe Sans.
+
+## Background
+
+Fluent v9 currently uses the font Segoe UI, which is a static font: a typeface produced as several different fonts, with each font representing a specific combination of width, weight, and style.
+
+Segoe Sans is a variable font, a single font file that includes an entire font family, allowing for many different variations in style without the need to use a separate file for each.
+
+## Problem statement
+
+<!--
+Why are we making this change? What problem are we solving? What do we expect to gain from this?
+
+This section is important as the motivation or problem statement is independent from the proposed change. Even if this RFC is not accepted this Motivation can be used for alternative solutions.
+-->
+
+The current font Segoe UI, a static font, denies designers of the flexibility they can get with a variable font. In addition, the baseline of Segoe UI is not true center, which requires adjustments in components on a case-by-case basis, which is time consuming.
+
+## Detailed Design or Proposal
+
+<!-- This is the bulk of the RFC. Explain the proposal or design in enough detail for the intended audience to understand. -->
+
+This RFC proposes that we switch the font used in the repo from Segoe UI to Segoe Sans.
+
+Because Segoe Sans is a variable font, the main concern is that content may shift significantly within existing components, and that layouts might need adjusting with isn't possible via theme.
+
+### Testing
+
+To compare Segoe Sans to Segoe UI, I used [odiff](https://github.com/dmtrKovalenko/odiff), a pixel-by-pixel image visual difference tool, to determine the visual differences in components using the two fonts.
+
+One example comparison can be seen below:
+
+> [![Font Comparisons](../assets/fonts-diff.png)](../assets/fonts-diff.png)
+>
+> - There is a shift horizontally, but the horizontal difference is not major enough to impact components and layouts.
```

Comment:
Could we explore if tweaking the `letter-spacing` can minimize the horizontal shift?
Even though the shift is small, I'm still concerned this might cause some noticeable changes for texts creating potential breaks on interfaces that we might not control.

If the `letter-spacing` tweak works, we could ship the font change on v9 without any concerns. And for future versions (v10 maybe?), we could simply remove the `letter-spacing` tweak and have the Segoe Sans as is.

What do you think?

Reply:
I tried changing the `letter-spacing`, but it looks like the biggest letter spacing difference between the two fonts happens with numbers, so changing the letter spacing evenly unfortunately won't fix the spacing differences.

Here is the closest example I could get, with `letter-spacing` value of `0.1px`:
<img width="1257" alt="image" src="https://github.com/user-attachments/assets/5e20a4ee-8e16-4bbf-b602-10ddbf714deb">